sys-boot/plymouth/plymouth-24.004.60.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson readme.gentoo-r1 flag-o-matic
+
+SRC_URI="https://dev.gentoo.org/~aidecoe/distfiles/${CATEGORY}/${PN}/gentoo-logo.png"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.freedesktop.org/plymouth/plymouth"
+else
+	SRC_URI="${SRC_URI} https://www.freedesktop.org/software/plymouth/releases/${P}.tar.xz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
+fi
+
+DESCRIPTION="Graphical boot animation (splash) and logger"
+HOMEPAGE="https://cgit.freedesktop.org/plymouth/"
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE="debug +drm +gtk +pango selinux freetype +split-usr +udev doc upstart-monitoring systemd"
+
+BDEPEND="
+	app-text/docbook-xsl-stylesheets
+	>=dev-build/meson-0.62
+	dev-libs/libxslt
```

Comment:
and probably this one too

Reply:
Could you elaborate on what is wrong with this part?

Reply:
BDEPEND is for things we *execute* during build, DEPEND is for things we need, but do not execute during build. This is a library and should therefore be a DEPEND. Same holds fort the stylesheets, I doubt there is any executable code there.

Reply:
for `app-text/docbook-xsl-stylesheets` a large portion of packages in the reverse dependency search were BDEPEND, so I thought it would be better to place it there.

The same applies for `dev-libs/libxslt`, I saw that quite a lot of packages have it as a BDEPEND only, though there are quite a lot that DEPEND on `libxslt` 

I've not looked enough to really know, I'll move these later today if you think they need to be DEPEND though

Reply:
I suppose it might be executing one of these scripts:
```
equery f dev-libs/libxslt
* Searching for libxslt in dev-libs ...
* Contents of dev-libs/libxslt-1.1.39:
/usr
/usr/bin
/usr/bin/i686-pc-linux-gnu-xslt-config
/usr/bin/x86_64-pc-linux-gnu-xslt-config
/usr/bin/xslt-config -> x86_64-pc-linux-gnu-xslt-config
/usr/bin/xsltproc
```

If so then BDEPEND is correct, if it only uses the library instead then DEPEND is correct.

Reply:
[`libxlst` is executed at build time only for docs](https://gitlab.freedesktop.org/plymouth/plymouth/-/blob/24.004.60/docs/meson.build?ref_type=tags&blame=0#L1) from what I can see, so it can probably go under the `doc` USE flag. As for `app-text/docbook-xsl-stylesheets` they're also used only for docs (from what I can tell) and so can also be a `BDEPEND`.

I'll put them under the `doc` USE flag but keep them in `BDEPEND`, let me know if you find that it needs changing :+1:.